fitwsarah_frontend/src/components/authentication/RoleBasedSwitch.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import { isRole } from './authUtils';
 import "../authentication/switch.css"
+import { useTranslation } from "react-i18next";
```

Comment:
Do we have to use this import in every one of our file to allow the localization to work?

Reply:
Yes you need to import in every files